env.wochenende.yml

```diff
@@ -1,9 +1,10 @@
 name: wochenende
 channels:
-  - r
-  - conda-forge
+  - https://repo.anaconda.com/pkgs/free
   - bioconda
+  - r
   - defaults
+  - conda-forge
```

Comment:
@colindaven, tested latest version of Wochenende pipeline 1.7.5 with updated conda env. No issues while running the pipeline with updated env file.